src/IceRpc.Protobuf/Internal/AsyncEnumerableExtensions.cs

```diff
@@ -6,19 +6,19 @@
 using System.Diagnostics;
 using System.IO.Pipelines;
 
-namespace IceRpc.Protobuf;
+namespace IceRpc.Protobuf.Internal;
 
 /// <summary>Provides an extension method for <see cref="IAsyncEnumerable{T}" /> to encode elements into a <see
 /// cref="PipeReader"/>.</summary>
-public static class AsyncEnumerableExtensions
+internal static class AsyncEnumerableExtensions
```

Comment:
The equivalent Slice class is part of the public API, why are we making this internal?

Reply:
In Slice, it's part of the public API only because the generated code uses this API. If it didn't (like here for Protobuf), we would make it internal as well.